src/gateway/environments/jsdom-sixteen/index.js

```diff
@@ -0,0 +1,12 @@
+// @flow
```

Comment:
I put this together because I envisage that long term, environments would probably be their own packages so that users of our `render-gateway` implementation can install only the one they want to use. I feel this keeps things clearly grouped in a manner that makes sense and also makes it easier to namespace this stuff in the final API we currently expose from the root gateway index

Reply:
Yeah, I like this! It'll make future JSDOM upgrades, or other environment experiments, a lot easier to coordinate, as well. Musing: I wonder if we might want a way to configure with env is used at request time (so that we could A/B test different environments, potentially).

Reply:
Yeah, there's nothing preventing a gateway implementation doing that with environments if it so chooses. It could, for example, create an environment that switches environments based on render.